src/main/java/net/shadowmage/ancientwarfare/npc/entity/faction/NpcFaction.java

```diff
@@ -163,7 +163,10 @@ public void setFactionNameAndDefaults(String factionName) {
 		this.factionName = factionName;
 		FactionNpcDefault npcDefault = NpcDefaultsRegistry.getFactionNpcDefault(this);
 		applyFactionNpcSettings(npcDefault);
+		// do not apply the default equipment if the hasCustomEquipment tag was set to true
+		if (!getCustomEquipmentOverride()) {
 		npcDefault.applyEquipment(this);

```

Comment:
Just fix the formatting here (should be indented) and that should be it for the PR

Reply:
Fixed